core/src/main/java/jenkins/BreadCrumbItem.java

```diff
@@ -0,0 +1,43 @@
+package jenkins;
+
+
+
+public class BreadCrumbItem {
+
+    private String displayName;
+    private boolean hasContextualMenu;
+    private BreadCrumb breadCrumb;
```

Comment:
Where is this assigned?

Reply:
I am not sure how to get this to show up in the jelly files.I included it so that it can be set and then used in a jelly file